discord/invite.py

```diff
@@ -151,7 +151,7 @@ def created_at(self):
 
     @property
     def icon_url(self):
-        """Returns the URL version of the guild's icon. Returns an empty string if it has no icon."""
+        """:class:`Asset`: Returns the Guilds's icon asset."""
```

Comment:
```suggestion
        """:class:`Asset`: Returns the guild's icon asset."""
```

Were you trying to refer to a ``:class:`Guild` `` instance? (Same for the rest)